contact.html

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Contact OME
+title: Contact the OMERO Team
 menu-id: l1-more
 description: Details on how to contact the OME and OMERO community and development teams with any questions, issues or problems.
```

Comment:
In the closed PR you had this sentence as:
'Details on how to contact the OMERO team and OME community and development teams with any questions, issues or problems.`
which I think needs to be here too.